migrations/002_add_app_password_to_accounts.py

```
@@ -0,0 +1,92 @@
+            result = conn.execute(text("PRAGMA table_info(email_accounts)"))
+            columns = [row[1] for row in result]
+
+            if 'app_password' not in columns:
+                print("Adding app_password column to email_accounts table...")
+                conn.execute(text(
+                    "ALTER TABLE email_accounts ADD COLUMN app_password VARCHAR(255)"
+                ))
+                conn.commit()
+                print("✅ Successfully added app_password column")
+            else:
+                print("ℹ️  app_password column already exists, skipping")
```

Ruff (0.13.3)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)